src/bindings/python/constraints.txt

```diff
@@ -10,10 +10,11 @@ pytest-timeout==2.3.1
 # Python bindings
 build<1.3
 pygments>=2.8.1
-setuptools>=65.6.1,<75.3.0
+setuptools>=70.1,<75.7
 sympy>=1.10
 wheel>=0.38.1
 patchelf<=0.17.2.1
+packaging<=22.0
```

Comment:
Why `<=`?. Shouldn't `packaging` be [higher](https://github.com/pypa/setuptools/issues/4483#issuecomment-2236339726)?

Reply:
Yes, good catch, fixed :)